backend/src/schema.graphql

```diff
@@ -310,3 +310,12 @@ type SharedInboxEndpoint {
   id: ID!
   uri: String
 }
+
+type SocialMedia {
+  id: ID!
+  uri: String
+}
+
+input urlInput {
+  url: String! @constraint(format: "uri")
```

Comment:
@mattwr18 nice! Does it work??

Reply:
no... you probably didn't see, but I wrote in my commit message the back end is not compiling with `Error: Unknown directive "constraint"`

I needed to go, so I pushed it up cause @Tirokk wanted to have a closer look